src/amo/sagas/reviews.js

```diff
@@ -60,6 +69,41 @@ function* fetchReviews({
   }
 }
 
+function* fetchRatingSummary({
+  payload: { errorHandlerId, addonId },
+}: FetchRatingSummaryAction): Generator<any, any, any> {
```

Comment:
Why is Flow not complaining here? `type` is required in the `FetchRatingSummaryAction` definition.

Reply:
This is just destructor syntax. It is safe and acceptable to only partially destruct an object.